### ISSUE TITLE
ADD ZoeDepth

### ISSUE THREAD

**Issue by itsmeterada** [axinc-ai/ailia-models#1376]
https://github.com/isl-org/ZoeDepth

**Comment by Akasan** [comment]
## メモ
- レポジトリのREADMEに従ってモデルを読み込もうとするとstate_dict周りでエラーが起きることがあったが、こちらの[イシュー](https://github.com/isl-org/ZoeDepth/issues/97)に従って特定のtimmをインストールすると解決

**Comment by kyakuno** [comment]
https://arxiv.org/abs/2302.12288

**Comment by kyakuno** [comment]
relative depth estimationではなく、metric depth estimationと呼ぶ。
